.storybook/middleware.js

```diff
@@ -9,6 +9,8 @@ const { createProxyMiddleware } = require('http-proxy-middleware');
 const retryNumber = 3;
 
 module.exports = function (app) {
+    // Set to true when you want to test CORS in Storybook
+    const isCorsEnabled = false;
```

Comment:
as we discussed on your PublickRepoFork PR, I think isCorsEnabled might be confusing variable name since you are referring if this feature is enabled on the backend, but here you use in a way to basically do cors related stuff if **not** enabled, lets change the name to "shouldProxy" or something.